src/ornament.ts

```diff
@@ -270,12 +298,12 @@ export class Ornament extends Modifier {
       if (this.delayXShift !== undefined) {
         delayXShift = this.delayXShift;
       } else {
-        delayXShift += this.glyph.getMetrics().width / 2;
+        delayXShift += (this.glyph.getMetrics().width ?? 0) / 2;
```

Comment:
This is a silent failure, check for `width` explicitly.

Reply:
resolved